### DISCUSSION TITLE
～～～

### DISCUSSION THREAD

**Discussion by liugunshi** [share121/inter-knot#800] [其他]
![mmexport1723259103197](https://github.com/user-attachments/assets/a896fd16-6da7-48ab-80f1-6d8d7e606819)
不会英语，发帖子好难

**Comment by liugunshi** [comment]
阿巴阿巴

**Comment by SDCOM-0415** [comment]
习惯了就好了，用久了就基本看的懂了

**Comment by AurEian** [comment]
你需要一个翻译插件

**Comment by Lephaethon** [comment]
其实你只需要在输入框输入就好(　ﾟ皿ﾟ)

**Comment by mhpmc** [comment]
edge浏览器自带的翻译挺好用的

**Comment by gaojiaobei** [comment]
这个委托我接了！（不是接委托脚本）
